Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "identicon-rs"
-version = "5.0.0"
+version = "5.0.1"
```

Comment:
If you drop this change, it looks ready to merge. For the moment, version bumps are intentionally their own commits for now that I follow up with. I want to automate cutting versions some more though through some CI/CD. I just gotta figure out the templating or automatic version bumps.

Reply:
that sounds quite annoying to make a separate commit for the version bump for every patch but ok. 

Can't you just create a script/github action that changes the version with regex & runs cargo publish?